Making minor change so that I can create a pull request for review

melissaryandigital opened the pull request:
Please review ./server/database/seedingScript.js

Gkolman reviewed (commented):
test test

Gkolman reviewed (commented):
Hey Melissa this looks awesome. Only thing i would suggest is maybe using a template literal on lines 41 and 38 and maybe trying to get in the habit of avoiding variable names like data, because sometimes it can be hard to rememer what it stands for if you haven't worked in the file in a while or someone like me is reading it and they're are multiple "data's" so i have to pay attention to scope closure.

melissaryandigital commented:
Thanks @Gkolman!  Not sure if you were reviewing server.js only or server and seedingScript.js (I had made a second PR to try to make sure seedingScript would appear).  Just a heads up, you can leave a comment directly on top of the line numbers in the code view.

I took your suggestion and updated to no longer use 'data' in my seeding script (I will update server.js as well soon).  And with that I also discovered a separate issue where my code was not working as expected due to using 'data' and scope closure in seedingScript.  So thank you for pointing that out!

Also, I have replaced the frankenstein query string in server.js with the axios.get option where you can specify the parameters and it creates the query string for you.
